nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -148,6 +148,24 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 - (Neo)Vim can not be configured with `configure.pathogen` anymore to reduce maintainance burden.
 Use `configure.packages` instead.
 
+- The WordPress module now stores themes and plugins in a directory name without appending the current version. This avoids loosing settings when upgrading themes and plugins as some of them use the directory name to store the settings in the database. This needs manual intervention in case of using such a broken plugin or theme which is rare.
+  ```bash
+  nix-shell -p wp-cli
+  cat /etc/httpd/httpd.conf  | grep "\-wordpress-"
+  cd /nix/store/xxx-wordpress-xxx/share/wordpress
+  sudo -u wordpress wp maintenance-mode activate
+  sudo -u wordpress wp maintenance-mode status
+  sudo -u wordpress wp db export
```

Comment:
This command also crashes with a weird error on NixOS currently. As you realize this you probably just backup using builtin database backup tools but as this is highly dependent on the exact database settings I don't think it is useful to provide another command here. Also I hope this gets fixed in wp-cli soon.